packaging/npm-publish.sh

```diff
@@ -39,12 +39,14 @@ $NIMIQ_VERSION
     "worker-js.js",
     "worker.js.map",
     "worker-wasm.js",
-    "worker-wasm.wasm"
```

Comment:
Doesn't this package.json also need a `"types": "types.d.ts",` entry?

Reply:
I think it's not necessary, since in this package the files are placed in the top level directory and thus typescript detects `types.d.ts` automatically. I added an entry to package.json anyway.